test/3.x/app.js

```diff
@@ -134,7 +134,7 @@ app.get('/escape', function(req, res) {
 app.get('/locals', function(req, res) {
   res.locals.person = 'Alan';
   res.render('locals', {
-    layout: false,
+    layout: 'layout_locals',
```

Comment:
This should cover both cases: with or without a layout. If `@person` does not work with a normal template (without layout), we wouldn't see "Alan" in "Alan has a kid named Jimmy" in this test.

But we can always leave the existing test as is and add a new one for the layout case. What do you think?